build-logic/src/main/kotlin/org/jetbrains/conventions/base-java.gradle.kts

```diff
@@ -22,6 +22,12 @@ java {
 tasks.withType<Test>().configureEach {
     useJUnitPlatform()
 
+    maxParallelForks = if (System.getenv("CI") != null) {
```

Comment:
Or we can read `dokka_integration_test_parallelism` like

https://github.com/Kotlin/dokka/blob/3ea414251d32108dab8b14d41a672e9834824690/build-logic/src/main/kotlin/org/jetbrains/conventions/dokka-integration-test.gradle.kts#L54-L56

Reply:
5facf7eee5f29f672345b1cae95d1fc98b9ad388